docs/reference/docs/update.asciidoc

```diff
@@ -124,13 +144,16 @@ the old. Setting `detect_noop` to `false` will cause Elasticsearch to always
 update the document even if it hasn't changed. For example:
 [source,js]
 --------------------------------------------------
-curl -XPOST 'localhost:9200/test/type1/1/_update' -d '{
+POST test/type1/1/_update
+{
     "doc" : {
         "name" : "new_name"
     },
-    "detect_noop": false
-}'
+    "detect_noop": true
```

Comment:
s/true/false/ I think. The paragraph talks about setting it to `false` and how the default is `true`.